### PR TITLE
Feature/fe/cr 138 hide onboarding GitHub feedback

### PR DESCRIPTION
**Pull request by pd-redis**

### CODE REVIEW

**Comment by nelsonwellsredis** [comment]
@pd-redis - looks like you missed the "Let us know what you think" link in the bottom right corner. Check out the `BottomGroupMinimized.tsx` component

**Comment by pd-redis** [comment]
> @pd-redis - looks like you missed the "Let us know what you think" link in the bottom right corner. Check out the `BottomGroupMinimized.tsx` component

Good catch, handled

**Review by nelsonwellsredis — APPROVED** [review]

File: redisinsight/ui/src/components/navigation-menu/NavigationMenu.tsx [hunk 1]
```diff
@@ -315,33 +315,47 @@ const NavigationMenu = () => {
...
-          position="right"
+        <FeatureFlagComponent
+          name={FeatureFlags.disabledByEnv}
+          enabledByDefault
+          otherwise={(
+            <Divider
+              color="transparent"
+              className="eui-hideFor--xs eui-hideFor--s"
+              variant="middle"
+            />
```
Comment (romansergeenkosofteq) [inline redisinsight/ui/src/components/navigation-menu/NavigationMenu.tsx:326]: Do we need this transparent Divider?

Comment (pd-redis) [reply redisinsight/ui/src/components/navigation-menu/NavigationMenu.tsx:326]: Kind of, if we don't have a divider at all, the sidebar jumps when you hover over the Help button, shows vertical and horizontal scrollbars.
If it is not transparent, then we get a divider that has nothing after it.
Maybe more elegant solution is possible but have to dig into how to disable the hover effect for the help button

**Review by romansergeenkosofteq — APPROVED** [review]

**Pull request merged**